job-service-unit-tests/src/test/java/com/hpe/caf/services/job/api/JobsPutTest.java

```diff
@@ -264,12 +272,14 @@ public void testCreateJob_Failure_TaskDataNotSpecified() throws Exception {
         job.setTask(action);
 
         //  Test failed run of job creation where task data has not been specified.
-        try {
-            JobsPut.createOrUpdateJob("partition", "067e6162-3b6f-4ae2-a171-2470b63dff00", job);
-        } catch (BadRequestException bre) {
-            assertEquals(JobsPut.ERR_MSG_TASK_DATA_NOT_SPECIFIED, bre.getMessage());
-            throw bre;
-        }
+        Assertions.assertThrows(BadRequestException.class, () -> {
```

Comment:
This seems like it might have been a good case where `.assertThrows()` could have been used properly.  I.e. instead of ignoring the result (and therefore requiring the warning), get the resultant exception and then make assertions on it.